diff -up kig-19.07.80/scripting/python_scripter.cc.omv~ kig-19.07.80/scripting/python_scripter.cc
--- kig-19.07.80/scripting/python_scripter.cc.omv~	2019-07-27 21:51:42.880183664 +0200
+++ kig-19.07.80/scripting/python_scripter.cc	2019-07-27 21:53:32.083626003 +0200
@@ -390,7 +390,7 @@ PythonScripter::PythonScripter()
   // the newstring stuff is to prevent warnings about conversion from
   // const char* to char*..
   char* s = newstring( "kig" );
-  PyImport_AppendInittab( s, initkig );
+  PyImport_AppendInittab( s, PyInit_kig );
   //  we can't delete this yet, since python keeps a pointer to it..
   // This means we have a small but harmless memory leak here, but it
   // doesn't hurt at all, since it could only be freed at the end of
